docs/SPEC.md

```diff
@@ -200,29 +200,34 @@ Examples:
     1mo5d // 1 month and 5 days
 
 Durations can be added to date times to produce a new date time.
-Adding and subtracting durations to date times normalizes the dates.
-For example July 32 converts to August 1.
 
 Addition and subtraction of durations to date times do not commute and are left associative.
-Addition and subtraction of durations to date times applies months, days and seconds in that order and then normalizes.
+Addition and subtraction of durations to date times applies months, days and seconds in that order.
+When months are added to a date times and the resulting date is past the end of the month, the day is rolled back to the last day of the month.
+
 
 Examples:
 
     2018-01-01T00:00:00Z + 1d       // 2018-01-02T00:00:00Z
     2018-01-01T00:00:00Z + 1mo      // 2018-02-01T00:00:00Z
-    2018-01-01T00:00:00Z + 2mo30d   // 2018-03-30T00:00:00Z
-    2018-01-01T00:00:00Z + 1mo30d   // 2018-03-02T00:00:00Z, February 30th is normalized to March 2 in 2018 since it is not a leap year.
+    2018-01-01T00:00:00Z + 2mo      // 2018-03-01T00:00:00Z
```

Comment:
Can there be a mention in here about one of the other comments you made? I found it interesting and it may not be immediately obvious the behavior. You had this:

```
first = 2018-01-31T00:00:00Z // January 31
second = first + 1mo // February 31st is rolled back to the last day of the month
third = second + 1mo // The duration keeps track that it was originally January 31 so this is March 31 again instead of March 28

You also say "February 31th" instead of "31st" on line 216.

Reply:
@jsternberg The above is no longer true, removing the rolling forward rule breaks this. There is no hidden state in dates. If you want that behavior you have to multiply the duration first.

first = 2018-01-31T00:00:00Z // January 31
second = first + 1mo // February 31st is rolled back to the last day of the month Feb 28th
third = first + 2*1mo // March 31st

Reply:
Sure.